stix_shifter_modules/reaqta/stix_transmission/connector.py

```diff
@@ -150,8 +150,8 @@ def modify_result(cls, data):
                 dict_list = []
                 for tactic in tactic_list:
                     
-                    if (f"{tactic}" in TACTIC_NAME_MAPPING):
-                        dict_list.append({"tactic_number": tactic, "tactic_name": TACTIC_NAME_MAPPING.get(f"{tactic}"), "technique":technique})
+                    if (f"{tactic}" in tactic_name_mapping):
+                        dict_list.append({"tactic_number": tactic, "tactic_name": tactic_name_mapping.get(f"{tactic}"), "technique":technique})
```

Comment:
is there a reason you need to f'string the tactic value?

Reply:
The mapping was in string, but the values were int. I've changed the mapping to match what the API responds with (int) so I won't need the f string.